src/Illuminate/Support/Collection.php

```diff
@@ -201,9 +201,19 @@ public function first(callable $callback = null, $default = null)
      *
      * @return static
      */
-    public function flatten()
+    public function flatten($depth = INF)
```

Comment:
I dunno that I agree that the phpdoc should be `int|float`, since it is certainly expected that you pass an `int`. The fact that `INF` is represented as a `float` internally is just an unfortunate implementation detail really :/ Doesn't bother me too much either way honestly (I've never read a doc block in my life :smile: ) but I wouldn't want the doc block to suggest that it's actually designed for floats.